docs/_sass/_base.scss

```diff
@@ -173,3 +173,24 @@ figcaption {
         }
     }
  }
+
+ .wrapper__code-examples {
+     & .wrapper__code-example {
+        margin-bottom: 64px;
+        &:last-of-type {
+            margin-bottom: 24px;
+        }
+     }
+    /* Overwrite default code-block styling */
+    & .code-example {
+        & pre, & .highlight {
+            background: transparent;
+        }
+    }
+    & .code-example.pass {
+        background-color: #f1fff1 !important;
```

Comment:
why is `!important` needed here and below?
if they're required, please leave a comment on why they're required. If not, remove.
Also, it would be awesome to have the color as a comment (e.g. `light red` or `light green`)